model_compression_toolkit/common/hardware_representation/op_quantization_config.py

```diff
@@ -0,0 +1,220 @@
+# Copyright 2022 Sony Semiconductors Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import copy
+from enum import Enum
+from typing import List
+
+class QuantizationMethod(Enum):
```

Comment:
There are two QuantizationMethod enums

Reply:
No... I moved it from QuantizationConfig. You think it should stay there?

Reply:
Please check again there is the same enum twice.
Line 20 and 201